src/components/bill_card/BillCard.jsx

```diff
@@ -36,7 +58,10 @@ const BillCard = () => {
 				<hr />
 
 				<div class="bill-item-vertical">
-					<p class="typo-subtext">You will save Rs 1000 on this order</p>
+					<p class="typo-subtext">
+						You will save Rs {bill.totalOriginalPrice - bill.totalPrice} on this
+						order
+					</p>
```

Comment:
You need to update class to className in this file.

Reply:
used className instead of class in new commit.